rocky/tools/viewsets.py

```diff
@@ -31,6 +38,9 @@ def indemnification(self, request, pk=None):
 
     @indemnification.mapping.post
     def set_indemnification(self, request, pk=None):
+        if not request.user.has_perm("tools.add_indemnification"):
+            raise PermissionDenied()
```

Comment:
Could we list/output the missing permission here? To aid in the correct setup for the used user?